tooling/nargo_cli/tests/compile_success_empty/impl_with_where_clause/src/main.nr

```diff
@@ -1,18 +1,19 @@
 
 fn main() {
-    let array: [Field; 3] = [1, 2, 3];
-    assert(array.eq(array));
+    // Test is temporarily disabled
```

Comment:
Do we have a followup issue for this? Looks to be #3409? Can we link it here just so we do not forget.

Reply:
Yes, I just made #3409 for it